styles/style.css

```diff
@@ -0,0 +1,67 @@
+* {
```

Comment:
Really great breakdown of CSS for different pages across different files with this one common CSS file for shared styles. You can observe the cascade effect as the more specific page's styles take precedent because they are imported after!